application.py

```diff
@@ -32,5 +32,14 @@ def contact():
     return render_template('contact.html')
 
 
+@application.route('/register')
+def register():
+    return render_template('register.html')
+
+
+@application.route('/header.html')
+def header():
+    return render_template('header.html')
+
```

Comment:
missing two blank lines. Please look at CI results next time.

Reply:
This issue was fixed already since I looked in the CI resut hence I did few commits to fix such issues - did you look on my last commit? I don't see any problem with CI results.